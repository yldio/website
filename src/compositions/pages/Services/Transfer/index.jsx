/* eslint-disable global-require */

import React from 'react';
import styled from 'styled-components';
import remcalc from 'remcalc';
import is, { isNot } from 'styled-is';

import Container from 'components/Container';
import Card from 'components/Card';
import { H2, Copy } from 'components/Typography';
import { Row } from '../AddCode';

import { Section } from '../styled';

const Bar = styled.div`
  background: #5e5cb2;
  width: ${remcalc(100)};
  height: ${remcalc(4)};
  margin-top: ${remcalc(15)};
  margin-bottom: ${remcalc(70)};
`;

const Wrapper = styled(Container)`
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  ${isNot('noMargin')`
    margin-top: ${remcalc(-140)};
  `}
  margin-bottom: ${remcalc(70)};

  ${is('noMargin')`
    padding: 0;
  `}
`;

const CardsWrapper = styled.div`
  background: #ffff;
  padding-top: ${remcalc(70)};
  padding-bottom: ${remcalc(70)};
  margin-top: ${remcalc(140)};
`;

const TransferSection = styled(Section)`
  padding: 0;
  padding-top: ${remcalc(140)};
  margin-top: 0px;
  background: #190533;
`;

const Transfer = () => (
  <TransferSection data-selector="services:tech">
    <Container>
      <H2 white slim>
        How we educate for knowledge transfer
        <Bar />
      </H2>
    </Container>
    <CardsWrapper>
      <Wrapper>
        <Card
          title="Private Training"
          points={[
            'Hands-on learning experience, in-person classes',
            'Tailoried training program delivered on-site to suit your needs',
            'Ensures your software engineers share a baseline',
          ]}
        />
        <Card
          title="Pairing"
          points={[
            '	Ongoing tutoring that covers all aspect of tech and product delivery ',
            'Seasoned YLD engineers and designers available to answer questions at all times',
            'Experienced supervisio for code review',
          ]}
        />
        <Card
          title="Certification"
          points={[
            '	Gain recognition for expertise and ability to employ best practices ',
            'Assess engineering candidates expertise and compare the results with your teamâ€™s performance',
          ]}
        />
      </Wrapper>
      <Container>
        <Row>
          <H2 slim>
            Our goal is never to just deliver but always to also upskill your
            whole team.
            <Bar />
          </H2>
          <Copy>
            We are an agile, flexible and passionate team consisting of
            strategy, design and engineering.
            <br />
            <br />
            We aspire to bring you the most up-to-date technology, with the
            minimal hassle possible.
            <br />
            <br />
            We take a hands on approach in our work, engaging directly with you
            in every stage of the development process.
            <br />
            <br />
            We take ownership of the delivery of the project or we can integrate
            at critical points with your team according to your needs.
            <br />
            <br />
            We enable your organisation to deliver world-class technology and
            user experiences, long after we leave.
          </Copy>
        </Row>
      </Container>
    </CardsWrapper>
  </TransferSection>
);

export default Transfer;
