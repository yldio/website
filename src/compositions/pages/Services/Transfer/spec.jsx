import React from 'react';
import { shallow } from 'enzyme';
import Transfer from './index';

test('compositions/pages/Services/Transfer snapshot', () => {
  const wrapper = shallow(<Transfer />);

  expect(wrapper).toMatchSnapshot();
});
