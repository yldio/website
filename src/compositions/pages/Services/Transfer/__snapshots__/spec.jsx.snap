// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/pages/Services/Transfer snapshot 1`] = `
<Transfer__TransferSection
  data-selector="services:tech"
>
  <Container>
    <Typography__H2
      slim={true}
      white={true}
    >
      How we educate for knowledge transfer
      <Transfer__Bar />
    </Typography__H2>
  </Container>
  <Transfer__CardsWrapper>
    <Transfer__Wrapper>
      <Card
        points={
          Array [
            "Hands-on learning experience, in-person classes",
            "Tailoried training program delivered on-site to suit your needs",
            "Ensures your software engineers share a baseline",
          ]
        }
        title="Private Training"
      />
      <Card
        points={
          Array [
            "	Ongoing tutoring that covers all aspect of tech and product delivery ",
            "Seasoned YLD engineers and designers available to answer questions at all times",
            "Experienced supervisio for code review",
          ]
        }
        title="Pairing"
      />
      <Card
        points={
          Array [
            "	Gain recognition for expertise and ability to employ best practices ",
            "Assess engineering candidates expertise and compare the results with your teamâ€™s performance",
          ]
        }
        title="Certification"
      />
    </Transfer__Wrapper>
    <Container>
      <AddCode__Row>
        <Typography__H2
          slim={true}
        >
          Our goal is never to just deliver but always to also upskill your whole team.
          <Transfer__Bar />
        </Typography__H2>
        <Typography__base>
          We are an agile, flexible and passionate team consisting of strategy, design and engineering.
          <br />
          <br />
          We aspire to bring you the most up-to-date technology, with the minimal hassle possible.
          <br />
          <br />
          We take a hands on approach in our work, engaging directly with you in every stage of the development process.
          <br />
          <br />
          We take ownership of the delivery of the project or we can integrate at critical points with your team according to your needs.
          <br />
          <br />
          We enable your organisation to deliver world-class technology and user experiences, long after we leave.
        </Typography__base>
      </AddCode__Row>
    </Container>
  </Transfer__CardsWrapper>
</Transfer__TransferSection>
`;
